PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+Fixes #issuenumber
+-
+
+Check everything which applies
+
+- [ ] I have added the issue number for which this pull request is created.
+
+-
```

Comment:
can you remove this `-`

Reply:
Sure no problem!

Reply:
Got it updated. Thank you very much for the review @arshadkazmi42 !

Reply:
You need to remove the `-` at line 8
At line 6 we want the `-`